src/rules/store/use-consistent-global-store-name.ts

```diff
@@ -20,6 +20,7 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
       category: 'Best Practices',
       description: 'Use a consistent name for the global store.',
       recommended: 'warn',
+      suggestion: true,
```

Comment:
Do you think it would be a good idea to adjust the README script to include when a rule has suggestions or not?

Reply:
Yea, it's on my backlog to add it to the generation of the rules 😅